src/stores/AccountController.ts

```diff
@@ -30,12 +31,13 @@ export const authNeedsRefresh = (auth: AuthData): boolean => {
 
 export const setJwtRefreshTimeout = () => {
   const auth = useAccountStore.getState().auth;
-  const { cleengSandbox } = useConfigStore.getState().getCleengData();
+
+  if (!auth?.refreshToken) return;
```

Comment:
Is this basically to skip this code for inplayer?  If so, can we at least add a comment here to that effect?